djangocms_snippet/admin.py

```diff
@@ -68,5 +69,31 @@ def has_delete_permission(self, request, obj=None):
             )
         return False
 
+    def has_change_permission(self, request, obj=None):
+        return False
+
+    def _get_preview_url(self, obj):
+        """
+        Return the change url which will be rendered in read only mode
+        :return: method or None
+        """
+        change_url = reverse(
```

Comment:
Does this go into readonly mode always? Or does it load the edit view when you own the draft?

Reply:
@Aiky30 The snippet view remains in read only so having another look at this